src/sentry/issues/json_schemas.py

```diff
@@ -0,0 +1,46 @@
+from typing import Any, Mapping
+
+EVENT_PAYLOAD_VERSIONS: Mapping[int, Mapping[str, Any]] = {
+    0: {
+        "type": "object",
+        "properties": {
+            # required properties
+            "event_id": {"type": "string", "minLength": 1},
+            "platform": {"type": "string", "minLength": 1},
+            "project_id": {"type": "integer"},
+            "tags": {"type": "object"},
+            "timestamp": {"type": "string", "format": "date-time"},
+            "message_timestamp": {"type": "string", "format": "date-time"},
+            # "title": {"type": "string", "minLength": 1}, leaving this out, for now
+            # non-required properties
+            "breadcrumbs": {"type": "object"},
+            "contexts": {"type": "object"},
+            "dist": {
+                "type": "object",
+                "properties": {
+                    "dist": {"type": "string", "minLength": 1},
+                    "release": {"type": "string", "minLength": 1},
```

Comment:
How is `dist.release` different from the top level `release` property?

Reply:
unclear haha. i'm just going off the spec. release is `Release version of the application` and dist is `The distribution of the application.`

see: https://develop.sentry.dev/sdk/event-payloads/